src/Features/Core/Portable/Completion/CompletionHelper.cs

```diff
@@ -289,23 +291,31 @@ private static int CompareExpandedItem(CompletionItem item1, PatternMatch match1
             }
 
             // We prefer expanded item over non-expanded one iff the expanded item 
-            // is an exact match whereas the non-expanded one isn't.
+            // is an exact match whereas the non-expanded one is worse than prefix match.
```

Comment:
you saw 'worse than', but you then use a != check as opposed to a `<` or `>` check.  is that desirable.